lib/web_ui/lib/src/engine/raw_keyboard.dart

```diff
@@ -96,6 +96,14 @@ class RawKeyboard {
     final FlutterHtmlKeyboardEvent event = FlutterHtmlKeyboardEvent(domEvent as DomKeyboardEvent);
     final String timerKey = event.code!;
 
+    // During IME composition, Tab fires twice (once for composition and once
+    // for regular tabbing behavior), which causes issues. Intercepting the
+    // tab keydown event during composition prevents these issues from occurring.
+    // https://developer.mozilla.org/en-US/docs/Web/API/Element/keydown_event#ignoring_keydown_during_ime_composition
+    if (event.type == 'keydown' && event.key == 'Tab' && event.isComposing) {
```

Comment:
Do we also need to ignore the corresponding keyup?

Reply:
It doesn't seem to affect the behavior whether I ignore it or not.  All of the conditional logic in in the keydown/keyup callback `_handleHtmlEvent` just seems to be handling for when the event type is `keydown`

Should I add the logic to ignore it regardless?

Reply:
If you tried it with an app, and the framework doesn't complain about it, then it should be fine.